x-pack/plugins/observability/common/utils/formatters/duration.test.ts

```diff
@@ -40,6 +40,30 @@ describe('duration formatters', () => {
     });
   });
 
+  describe('asTransactionRate', () => {
+    it('displays the not available label when the number is not finite', () => {
+      expect(asTransactionRate(Infinity)).toBe('N/A');
+      expect(asTransactionRate(-Infinity)).toBe('N/A');
+      expect(asTransactionRate(null)).toBe('N/A');
+      expect(asTransactionRate(undefined)).toBe('N/A');
+    });
+
+    it('displays the correct label when the number is finite', () => {
```

Comment:
Should numbers with more than one thousand separators (like (1,000,000,000)? 
Does it make sense here to test also negative numbers?

Reply:
That's a great point. I was going to address negative numbers in a separate PR cause it would not just add test coverage, but also introduce functional changes (does it make sense to show a negative rate?)

Reply:
Makes sense!